app/pdl.yaml

```diff
@@ -23,6 +23,8 @@ tags:
   description: "Title I Subtitle D Dairy Margin Coverage data."
 - name: "Title I: Commodities - Subtitle E, Supplemental Agricultural Disaster Assistance"
   description: "Title I Subtitle E Supplemental Agricultural Disaster Assistance data."
+- name: "Title I: Commodities - ARC/PLC"
```

Comment:
We might not need a new tag as these new endpoints fall under the tag "Title I: Commodities - Subtitle A", which is what is being currently used. These two lines could be removed.

Reply:
Resolving the conversation as this has been addressed. Thanks.